spec/v0.4.1-draft/spec.md

```diff
@@ -465,7 +465,7 @@ It should be rendered as a straight line, with optional direction markers at the
   The color of the arrow. Default is white (`#FFFFFF`). Inspired from SVG `stroke`.
 
 - **start**:
-  The start point of the arrow. The array contains the x and y coordinates.  
```

Comment:
Here, the two spaces are intentional to force a line break.  I got this from here https://www.markdownguide.org/basic-syntax/#line-break-best-practices

Reply:
We can also force a line break using ` \` (backslash at end of line) instead of 2 spaces, which pleases formatters and produces the same line break in the rendered version.